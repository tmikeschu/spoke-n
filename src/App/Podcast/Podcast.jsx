import React from "react"
import { Link } from "react-router-dom"

const Podcast = () => (
  <article className="Podcast">
    <p>
      We are producing a podcast as we pedal south. We are aiming to release an
      episode every 2 weeks, this, of course, depends on where in the world we
      are and whether or not we are able to upload. We'll be covering everything
      from local culture and interviews, to gear reviews and mechanical mishaps,
      to personal diatribes and shoutouts to Grandma.
      <br />
      <br />
      Episodes of The Spoken Tour will be available for download via iTunes,
      SoundCloud, Stitcher and a couple additional sites. We hope our maiden
      voyage into journalism doesn't totally suck, and if you listeners out
      there have any tips to help make that happen we would love to hear 'em.
      Leave us comments about the show on our <Link to="contact">
        Contact
      </Link>{" "}
      page.
    </p>
    <iframe
      title="Podcast"
      width="100%"
      height="600"
      scrolling="no"
      frameBorder="no"
      src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/users/290948600&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false&amp;visual=true"
    />
  </article>
)

export default Podcast
