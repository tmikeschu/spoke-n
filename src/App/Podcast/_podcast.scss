.podcast {
  background-color: rgba(0,0,0,.8);
  max-width: 1000px;
  margin: 1% auto;
  padding-bottom: 5em;
  text-align: center;
  p {
    background-color: $blue;
    color: #fff;
    text-align: center;
    font-size: 3em;
    margin-bottom: 10%;
    padding: 1% 0;
    margin-top: 1%;
  }
  iframe {
    width: 99%;
    background-image: url("../../public/loading.gif");
    background-repeat: no-repeat;
    background-position: 50% 50%;
    background-size: 20%;
  }
  @media (max-width: 640px) {
    width: 99%;
  }
}
