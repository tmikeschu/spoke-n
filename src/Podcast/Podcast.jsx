import React, { Component } from 'react';
import '../stylesheets/App.css';

export default class Podcast extends Component {
  render() {
    return (
      <article className="podcast">
        <p>Coming soon!</p>
        <iframe width="100%" height="600" scrolling="no" frameBorder="no" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/users/290948600&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false&amp;visual=true"></iframe>
      </article>
    );
  }
}
