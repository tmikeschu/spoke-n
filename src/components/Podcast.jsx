import React, { Component } from 'react';
import { Link } from 'react-router';
import '../stylesheets/App.css';

export default class Podcast extends Component {
  render() {
    return (
      <article className="podcast">
        <p>We are producing a podcast as we pedal south.  We are aiming to release an episode every 2 weeks, this, of course, depends on where in the world we are and whether or not we are able to upload. We'll be covering everything from local culture and interviews, to gear reviews and mechanical mishaps, to personal diatribes and shoutouts to Grandma.  Episodes of The Spoken Tour will be available for download via iTunes, SoundCloud, Stitcher and a couple additional sites.  We hope our maiden voyage into journalism doesn't totally suck, and if you listeners out there have any tips to help make that happen we would love to hear 'em.  Leave us comments about the show on our <Link to="contact">Contact</Link> page.</p>
        <iframe width="100%" height="600" scrolling="no" frameBorder="no" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/users/290948600&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false&amp;visual=true"></iframe>
      </article>
    );
  }
}
