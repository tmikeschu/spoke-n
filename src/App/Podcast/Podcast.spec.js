import React from 'react'
import { shallow } from "enzyme"
import Podcast from "./Podcast"
import { Link } from 'react-router'

describe("<Podcast />", () => {
  const podcast = shallow(<Podcast />)

  it("renders without crashing", () => {
    expect(podcast).toBeTruthy()
  })

  it("renders the correct content", () => {
    expect(podcast.equals(
      <article className="podcast">
        <p>We are producing a podcast as we pedal south.  We are aiming to release an episode every 2 weeks, this, of course, depends on where in the world we are and whether or not we are able to upload. We'll be covering everything from local culture and interviews, to gear reviews and mechanical mishaps, to personal diatribes and shoutouts to Grandma.  Episodes of The Spoken Tour will be available for download via iTunes, SoundCloud, Stitcher and a couple additional sites.  We hope our maiden voyage into journalism doesn't totally suck, and if you listeners out there have any tips to help make that happen we would love to hear 'em.  Leave us comments about the show on our <Link to="contact">Contact</Link> page.</p>
        <iframe title="Podcast" width="100%" height="600" scrolling="no" frameBorder="no" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/users/290948600&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false&amp;visual=true"></iframe>
      </article>
    )).toBeTruthy()
  })

  describe("snapshot", () => {
    it("is valid", () => {
      expect(podcast.getNodes()).toMatchSnapshot()
    })
  })
})
