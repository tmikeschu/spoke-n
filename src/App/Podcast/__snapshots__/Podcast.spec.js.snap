// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Podcast /> snapshot is valid 1`] = `
<article
  className="Podcast"
>
  <p>
    We are producing a podcast as we pedal south.  We are aiming to release an episode every 2 weeks, this, of course, depends on where in the world we are and whether or not we are able to upload. We'll be covering everything from local culture and interviews, to gear reviews and mechanical mishaps, to personal diatribes and shoutouts to Grandma.
    <br />
    <br />
    Episodes of The Spoken Tour will be available for download via iTunes, SoundCloud, Stitcher and a couple additional sites.  We hope our maiden voyage into journalism doesn't totally suck, and if you listeners out there have any tips to help make that happen we would love to hear 'em.  Leave us comments about the show on our 
    <Link
      replace={false}
      to="contact"
    >
      Contact
    </Link>
     page.
  </p>
  <iframe
    frameBorder="no"
    height="600"
    scrolling="no"
    src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/users/290948600&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&visual=true"
    title="Podcast"
    width="100%"
  />
</article>
`;
